taxi/models.py

```diff
@@ -0,0 +1,36 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    class Meta:
+        ordering = ("name",)
+
+    def __str__(self):
+        return f"{self.name}"
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=255, unique=True, )
+
+    class Meta:
+        ordering = ("username",)
+
+    def __str__(self):
+        return f"{self.username} ({self.license_number})"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(Manufacturer, on_delete=models.CASCADE)
+    drivers = models.ManyToManyField(Driver, related_name="cars")
+
+    class Meta:
+        ordering = ("model",)
+
+    def __str__(self):
+        return f"{self.model}"
```

Comment:
Avoid overusing f-strings when they are not necessary. Instead of `return f"{self.model}"`, you can simply use `return self.model`.